types/screeps/screeps-tests.ts

```diff
@@ -354,7 +369,7 @@ interface CreepMemory {
     const interShardData = JSON.parse(RawMemory.interShardSegment);
     if (interShardData.creeps[creep.name]) {
         creep.memory = interShardData[creep.name];
-        delete interShardData.creeps[creep.name]; // tslint:disable-line no-dynamic-delete
+        delete interShardData.creeps[creep.name];
```

Comment:
The comment is required to prevent TSLint from failing the build.
It might be worth considering to remove the entire line, I'm not quite sure what it's supposed to test anyways.